src/allotropy/parsers/utils/values.py

```diff
@@ -38,6 +39,19 @@ def try_float(value: str, value_name: str) -> float:
         raise AllotropeConversionError(msg) from e
 
 
+def try_non_nan_float(value: str) -> float:
```

Comment:
@alejandro-salgado is this util function used anywhere? I only see the definition.

Reply:
It is an alternative to try_float in which an error is raised if a nan value is read.